.github/workflows/spotless.yml

```
@@ -0,0 +1,29 @@
+name: Spotless Check
+
+on:
+  push:
+    branches: [ "**" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)